community/kernel/src/main/java/org/neo4j/kernel/impl/store/record/AbstractBaseRecord.java

```diff
@@ -31,6 +31,9 @@
 public abstract class AbstractBaseRecord implements CloneableInPublic
 {
     private long id;
+    // Used for the "record unit" feature where one logical record may span two physical records,
+    // as to still keep low and fixed record size, but support occasionally bigger records.
+    private long secondaryId;
```

Comment:
that's true, and all of those formats is _dynamic_ formats, and my proposal is/was to have a subclass of  `AbstractBaseRecord` lets say, `DynamicBaseRecord` with that field.